### ISSUE TITLE
nextjs-keystone-demo isn't updated

### ISSUE THREAD

**Issue by JamBalaya56562** [keystonejs/keystone#9410]
## Description
[Keystone + Next.js Demo](https://nextjs-keystone-demo.vercel.app/) has a below invalid link.

> [Check out the example in the repo more info.](https://github.com/keystonejs/keystone/tree/main/examples/nextjs-keystone)

However, [framework-nextjs-pages-directory](https://github.com/keystonejs/keystone/tree/main/examples/framework-nextjs-pages-directory) and [framework-nextjs-app-directory](https://github.com/keystonejs/keystone/tree/main/examples/framework-nextjs-app-directory) show a correct link.

> [Check out the example in the repo more info.](https://github.com/keystonejs/keystone/tree/main/examples/framework-nextjs-app-directory)

In conclusion, I want to update [Keystone + Next.js Demo](https://nextjs-keystone-demo.vercel.app/) to latest example.

**Comment by dcousens** [comment]
@flexdinesh what repository is feeding that example?

**Comment by flexdinesh** [comment]
@dcousens looks like the examples were renamed in this change 333152e620183f310be892f1c82fbf847b47ecae. This is the updated link now - https://github.com/keystonejs/keystone/tree/main/examples/framework-nextjs-two-servers.

**Comment by JamBalaya56562** [comment]
Thank you for nice investigation.
My description is sufficient.

At first, I wanted to use Keystone with Next.js.
Then I found [Use Keystone in Next.js applications](https://keystonejs.com/blog/nextjs-keystone) blog ([source code](https://github.com/keystonejs/keystone/blob/main/docs/content/blog/nextjs-keystone.md)).
Next I went to [Keystone + Next.js Demo](https://nextjs-keystone-demo.vercel.app/) to try it out with below link.

![image](https://github.com/user-attachments/assets/336acbb7-a047-4ffc-bdcf-7d86be5dca58)

However, the demo has invalid link.
I'm afraid that it is different from the current examples.

I want to confirm which example is same with [Keystone + Next.js Demo](https://nextjs-keystone-demo.vercel.app/).
